contracts/minter-suite/MinterFilter/MinterFilterV1.sol

```diff
@@ -55,6 +55,11 @@ contract MinterFilterV1 is IMinterFilterV0 {
     /// minter address => is an approved minter?
     mapping(address => bool) public isApprovedMinter;
 
+    modifier onlyNonZeroAddress(address _address) {
+        require(_address != address(0), "Must input non-zero address");
+        _;
+    }
```

Comment:
nit: can we add test coverage for this?

Reply:
Adding coverage to catch this in the `MinterFilterV0V1Views` tests.

Reply:
Eh–actually adding a new separate V1-specific `MinterFilterV1Views` test for this, it doesn't apply to V0.